server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/create_table_parquet_vpc-1.0.0.sql

```diff
@@ -0,0 +1,75 @@
+ --- DDL VPC create table statement definition
```

Comment:
should we include the integration name in the file name? I think for all of the pending PRs, we are just named it like `create_table-1.0.0`